sched/sched/sched_cpuload_oneshot.c

```diff
@@ -225,6 +231,37 @@ static void nxsched_oneshot_callback(FAR struct oneshot_lowerhalf_s *lower,
   nxsched_oneshot_start();
 }
 
+#ifdef CONFIG_PM
+static void nxsched_oneshot_pmnotify(struct pm_callback_s *cb, int domain,
```

Comment:
```suggestion
static void nxsched_oneshot_pmnotify(FAR struct pm_callback_s *cb, int domain,
```
FAR is required here, since this file is common to all targets.